ci-operator/step-registry/netobserv/install-operator/netobserv-install-operator-commands.sh

```diff
@@ -0,0 +1,207 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    openshift.io/cluster-monitoring: "true"
+  name: openshift-netobserv-operator
+EOF
+}
+
+create_og() {
+  cat <<EOF | oc apply -f -
+---
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: netobserv-operator
+  namespace: openshift-netobserv-operator
+spec:
+  upgradeStrategy: Default
+EOF
+}
+
+subscribe(){
+    cat <<EOF | oc apply -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: netobserv-operator
+  namespace: openshift-netobserv-operator
+spec:
+  channel: ${CHANNEL}
+  name: netobserv-operator
+  source: ${CATALOG_NAME}
+  sourceNamespace: openshift-marketplace 
+EOF
+}
+
+patch_csv_images(){
+  CSV=$(oc get csv -n openshift-netobserv-operator | grep -iE "net.*observ" | awk '{print $1}')
+
+  # patch as Downstream to scrape metrics
+  oc patch csv/$CSV -n openshift-netobserv-operator --type='json' -p='[{"op": "replace", "path": "/spec/install/spec/deployments/0/spec/template/spec/containers/0/env/3/value", "value": "true"}]'
+
+  if [[ $PATCH_EBPFAGENT_IMAGE ]]; then
+    PATCH="[{\"op\": \"replace\", \"path\": \"/spec/install/spec/deployments/0/spec/template/spec/containers/0/env/0/value\", \"value\": \"$PATCH_EBPFAGENT_IMAGE\"}]"
+      oc patch csv/$CSV -n openshift-netobserv-operator --type='json' -p="$PATCH"
+  fi 
+
+  if [[ $PATCH_FLOWLOGS_IMAGE ]]; then
+      PATCH="[{\"op\": \"replace\", \"path\": \"/spec/install/spec/deployments/0/spec/template/spec/containers/0/env/1/value\", \"value\": \"$PATCH_FLOWLOGS_IMAGE\"}]"
+      oc patch csv/$CSV -n openshift-netobserv-operator --type='json' -p="$PATCH"
+  fi
+
+  if [[ $PATCH_CONSOLE_PLUGIN_IMAGE ]]; then
+      PATCH="[{\"op\": \"replace\", \"path\": \"/spec/install/spec/deployments/0/spec/template/spec/containers/0/env/2/value\", \"value\": \"$PATCH_CONSOLE_PLUGIN_IMAGE\"}]"
+      oc patch csv/$CSV -n openshift-netobserv-operator --type='json' -p="$PATCH"
+  fi
+
+  if [[ $PATCH_OPERATOR_IMAGE ]]; then
+      PATCH="[{\"op\": \"replace\", \"path\": \"/spec/install/spec/deployments/0/spec/template/spec/containers/0/image\", \"value\": \"$PATCH_OPERATOR_IMAGE\"}]"
+      oc patch csv/$CSV -n openshift-netobserv-operator --type='json' -p="$PATCH"
+  fi
+}
+
+if [[ "$CATALOG_SOURCE" == "source" ]]; then
+    echo "====> Using upstream catalog bundle with tag: vmain"
+    CATALOG_IMAGE="quay.io/netobserv/network-observability-operator-catalog:v0.0.0-main"
+    CATALOG_NAME="netobserv-main"
+    CHANNEL="latest" # for upstream catalog source "latest" is the channel name.
+    deploy_catalog
+fi
+
+NAMESPACE=netobserv
+oc new-project ${NAMESPACE} || true
+
+echo "====> Deploying 0-click Loki"
+oc apply -f https://raw.githubusercontent.com/netobserv/documents/main/examples/zero-click-loki/1-storage.yaml -n ${NAMESPACE}
+oc apply -f https://raw.githubusercontent.com/netobserv/documents/main/examples/zero-click-loki/2-loki.yaml -n ${NAMESPACE}
```

Comment:
Not sure if we should rely on urls there as all the others yamls are directly inside this script :thinking: 

It could break all the PRs tests if something is introduced in these but on the other side it's ensuring the documented yamls are working fine !

Reply:
yes, we've been using this URLs across both our frontend testing for 0-click-loki, it helps to have config in one place as compared to having yamls at multiple places.